WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -224,10 +228,12 @@ public function is_targetted_token( $stackPtr ) {
 			}
 
 			if ( preg_match( $group['regex'], $token_content ) === 1 ) {
-				$this->process_matched_token( $stackPtr, $groupName, $token_content );
+				$skip_to = $this->process_matched_token( $stackPtr, $groupName, $token_content );
```

Comment:
> Shouldn't this check the previous value of $skip_to, and only overwrite it if the new value is less?

Good question. Should the highest or the lowest value be returned though ? A case can be made for either one.
Code wise, the simplest way would be to collect all the returned values in an array and return it at the end with either a `min()` or `max()` around it.

In practice, none of the sniffs which use the function restrictions abstract or the child abstracts of this one, return a value at this moment, but it could become an issue if any of them would at some point, though only if within that same sniff they would sniff for the same function in two different groups.
The only sniff which at this moment does so, is the `WP.AlternativeFunctions` sniff, which sniffs for `file_get_contents()` in both the `file_get_contents` and the `file_system_read` group.